src/systemctl/systemctl-list-units.c

```diff
@@ -405,14 +415,8 @@ static int output_sockets_list(struct socket_info *socket_infos, unsigned cs) {
         if (r < 0)
                 return r;
 
-        if (arg_legend != 0) {
-                printf("\n%s%u sockets listed.%s\n",
-                       cs > 0 ? ansi_highlight() : ansi_highlight_red(),
-                       cs,
-                       ansi_normal());
-                if (!arg_all)
-                        printf("Pass --all to see loaded but inactive sockets, too.\n");
-        }
+        if (arg_legend != 0)
```

Comment:
Might as well move this into the function too…

Reply:
I wanted to keep it explicit that the output of legend is optional. But I don't care much either way...